File: app/src/main/java/com/artmurka/artmurkaapp/Model/FormOrder/Success.java
package com.artmurka.artmurkaapp.Model.FormOrder;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Success {


    @SerializedName("msg")
    @Expose
    private final ThreadLocal<String> msg = new ThreadLocal<>();

    public String getMsg() {
        return msg.get();
    }

    public void setMsg(String msg) {
        this.msg.set(msg);
    }
}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/formorder/Success.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.formorder


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Success {


    @SerializedName("msg")
    @Expose
    private val msg = ThreadLocal<String>()

    fun getMsg(): String {
        return msg.get()
    }

    fun setMsg(msg: String) {
        this.msg.set(msg)
    }
}
